setup.py

```diff
@@ -43,7 +42,8 @@
     ],
     include_package_data=True,
     install_requires=[
-
+        'django-formtools>=2.0',
```

Comment:
the primary change is that now `pip install wizard_builder` should also install form tools and model utils (it didn't prior)

it's also a bit slower because `setuptools` is piping (heh) the wizard builder install instructions to `pip` (via `setup.py`) rather than us giving them to `pip` directly (via `requirements.txt`). but only a bit slower. 10 seconds maybe?